CDCgov/prime-reportstream
# Updates to Mac/M1 setup documentation

This PR intends to update the setup documentation based on the experiences of new team members over the past few weeks, specifically around macOS and M1 processors. Seems to have applied formatting settings from my IDE, sorry if that's annoying!

Test Steps:
1. Follow the latest docs on M1 Mac to verify that after a run of `./cleanslate.sh` these steps lead to a successful setup.c

## Changes

- Add `docker-compose up --detach vault 1>/dev/null 2>/dev/null` step to save the vault credentials after starting services in the M1 instructions.
- Show `openjdk@11`/`openjdk@${VERSION?}` in the examples for openjdk installation on macOS
- Add `echo export "JAVA_HOME=\$(/usr/libexec/java_home)" >> ~/.zshrc` step to the openjdk installation instructions for macOS

## Checklist

Doc only change, so most of these checks aren't relevant, but I went through the bulk of this setup again to make sure it worked out alright. As new members continue to join, we'll have additional feedback on the latest steps.

### Testing

### Process

## Linked Issues
- One-off PR for a doc update, no linked issue
